ModularTegustation/tegu_mobs/apocalypse_bird.dm

```diff
@@ -494,6 +508,22 @@
 		if(M.z == z && M.client)
 			flash_color(M, flash_color = "#CCBBBB", flash_time = 50)
 	addtimer(CALLBACK(GLOBAL_PROC, .proc/show_global_blurb, 5 SECONDS, "A long time ago, in a warm and dense forest lived three happy birds.", 25))
+	COOLDOWN_START(src, take_bird, 10 SECONDS)
+	force_bird_time = world.time + 3 MINUTES
+
+/mob/living/simple_animal/forest_portal/Life()
+	if(COOLDOWN_FINISHED(src, take_bird))
+		var/list/range_area = orange(1, src)
+		if(force_bird_time < world.time)
+			range_area = GLOB.mob_list // If they take too long, we shove them in MANUALLY.
```

Comment:
It would be a better idea to use the list of abnormalities instead.

Reply:
If there isn't one, then use simple mob list or create a new global list for abnormalities.